galasa-ui/src/tests/test-runs.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import { render, screen } from '@testing-library/react';
+import TestRunsPage from '@/app/test-runs/page';
+
+// Mock component dependencies
+jest.mock('@/components/PageTile', () => ({
+  __esModule: true,
+  default: ({ title }: { title: string }) => <div data-testid="page-tile">{title}</div>,
+}));
+
+jest.mock('@/components/common/BreadCrumb', () => ({
+  __esModule: true,
+  default: () => <div data-testid="breadcrumb">BreadCrumb</div>,
+}));
+
+jest.mock('@/components/test-runs/TestRunsContent', () => ({
+  __esModule: true,
+  default: () => <div data-testid="test-runs-content">Test Runs Content</div>,
+}));
+
+// Mock router
+const mockRouter = {
+  refresh: jest.fn(),
+};
+
+jest.mock('next/navigation', () => ({
+  useRouter: jest.fn(() => mockRouter),
+}));
```

Comment:
The `mockRouter` and `next/navigation` mocks can be removed since the `TestRunsPage` doesn't use the `useRouter` hook anywhere.